docs/advanced/catkin_shell_verbs.rst

```diff
@@ -0,0 +1,36 @@
+Shell support in ``catkin`` command
+===================================
+
+When you source following file, you can use bash/zsh wrapper function for shell support.
```

Comment:
"When you ``source`` the following file, you can use bash/zsh shell functions which provide added utility."?